### DISCUSSION TITLE
MOD + mouse click/drag ?

### DISCUSSION THREAD

**Discussion by mckellygit** [wezterm/wezterm#1173] [Q&A] [answered]
Hi,

Thank you for wezterm.
Does wezterm, or can I configure it to, send modifiers with the mouse events ?
In tmux I use ctrl and ctrl+alt modifiers for mouse click, double-click, triple-click and drag, but I cannot seem to see these events when running in wezterm.
Are there actions for sending all these type of events that I can add to my config ?

thx,
-m

**Comment by mckellygit** [comment]
ok, I will forget about this one and try to use wezterm alone.

**Reply by wez** [reply]
Sorry I didn't respond sooner; wezterm does send information about modifiers when mouse reporting is enabled, but I'm not sure how those translate through tmux.

**Comment by mckellygit** [comment]
Hi,
No problem.  
I discovered MODs with left click, etc. do get propagated through when using X.
Its when I have enable_wayland=true that I no longer see these events in tmux.
thx again,
-m

**Reply by wez** [reply]
I think the wayland modifier issue was fixed by https://github.com/wez/wezterm/commit/455ea89fa825ea8c0ffb3ff231777ab4b7bc2d51#diff-5c84495571a183dd167fae7ab4620731d21bb6e79267f0a89d3dfb3f1f810810 and should be working in the nightly builds.

**Comment by mckellygit** [comment, marked as answer]
ok, awesome.
thx for wezterm!
-m

**Comment by mckellygit** [comment]
This has been answered above.
thx,
-m
